### ISSUE TITLE
Fails with `Error: missingKey("url")` with Xcode 13.3.1

### ISSUE THREAD

**Issue by a-szotyori** [unsignedapps/swift-create-xcframework#63]
Since Xcode 13.3.1 the swift package manifest parser is broken. With Xcode 13.2 it works. The only error that's printed is: `Error: missingKey("url")`.

I investigated and found that manifest json is created using Swift 5.6 (due to Xcode 13.3.1), but `swift-create-xcframework` uses SwiftPM version 5.5 which can't parse the manifest json.

I updated the SwiftPM to 5.6, but it contains many breaking changes which requires refactoring of `swift-create-xcframework`...

**swift-create-xcframework version:** 
`2.1.0` or commit hash `a8376e964c5e29dae0290f53c3aa865bb5f34d00`
**Swift version:** 
```
swift-driver version: 1.45.2 Apple Swift version 5.6 (swiftlang-5.6.0.323.62 clang-1316.0.20.8)
Target: x86_64-apple-macosx12.0
```
**Environment:** 
Xcode 13.3.1 and macOS 12.3

### Checklist
- [ x] If possible, I've reproduced the issue using the `master` branch of this package
- [ x] I've searched for [existing GitHub issues](https://github.com/unsignedapps/swift-create-xcframework/issues)

### Steps to Reproduce

* Check out https://github.com/gini/gini-mobile-ios
* `$ cd BankSDK/GiniBankSDK`
* `$ swift create-xcframework`

### Expected behavior

An XCFramework is generated.

### Actual behavior

`Error: missingKey("url")` is printed.

The error is thrown on this line caused by `ManifestLoader.loadRootManifest()` completing with a failure: https://github.com/unsignedapps/swift-create-xcframework/blob/main/Sources/CreateXCFramework/PackageInfo.swift#L88

Here's the command that was used to parse the swift package manifest (I printed the `cmd` var from [here](https://github.com/apple/swift-package-manager/blob/release/5.5/Sources/PackageLoading/ManifestLoader.swift#L821)):
```
(lldb) po cmd
▿ 21 elements
  - 0 : "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc"
  - 1 : "-L"
  - 2 : "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm/ManifestAPI"
  - 3 : "-lPackageDescription"
  - 4 : "-Xlinker"
  - 5 : "-rpath"
  - 6 : "-Xlinker"
  - 7 : "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm/ManifestAPI"
  - 8 : "-target"
  - 9 : "x86_64-apple-macosx11.0"
  - 10 : "-swift-version"
  - 11 : "5"
  - 12 : "-I"
  - 13 : "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm/ManifestAPI"
  - 14 : "-sdk"
  - 15 : "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.3.sdk"
  - 16 : "-package-description-version"
  - 17 : "5.5.0"
  - 18 : "/Users/alpar/Documents/Projects/gini-mobile-ios/BankSDK/GiniBankSDK/Package.swift"
  - 19 : "-o"
  - 20 : "/var/folders/l8/6nfk4h5s7cl113d6qn6rs_2r0000gn/T/TemporaryDirectory.wzH6IV/ginibanksdk-manifest"
```

And the resulting parsed manifest json was: 
[parsed-manifest.json.txt](https://github.com/unsignedapps/swift-create-xcframework/files/8532434/parsed-manifest.json.txt)

The error was then thrown in SwiftPM 5.5 here: https://github.com/apple/swift-package-manager/blob/release/5.5/Sources/PackageLoading/PackageDescription4Loader.swift#L363

**Comment by jeffzheng0505** [comment]
I also encountered this problem in xcode13.3, Can somebody know how to resolve this problem?

**Comment by a-szotyori** [comment]
@jeffzheng0505 The solution we opted for was to install Xcode 13.2 (we found 
[this article](https://medium.com/a-swift-misadventure/how-to-install-multiple-versions-of-xcode-on-the-same-macos-machine-a2836387e57f) helpful)
and then use [xcode-select](https://riptutorial.com/xcode/example/19193/switching-command-line-tools-with-xcode-select) to make Xcode 13.2 the default while building the XCFramework with. Afterwards we reset the command line tools to the latest Xcode.

Our testing showed that the XCFrameworks built with Xcode 13.2 worked without issues when using with the latest Xcode version.

**Comment by jeffzheng0505** [comment]
I‘ve fixed the swift-create-xcframework's issue in XCode 13.3 and raised a pr. hope can be accepted!

**Comment by mohit-kapadia-at-lirisoft** [comment]
Still see the issue in Xcode 13.4.1. I know the fix is released in v2.2.0. But it is not working.

`192:CompanyKit lirisoft$ swift create-xcframework
Error: missingKey("url")
192:CompanyKit lirisoft$ swift-create-xcframework --version
2.2.0`

**Comment by a-szotyori** [comment]
Thank you @sapharow for fixing the issue. I can confirm that the fix worked for our [repository](https://github.com/gini/gini-mobile-ios) now using Xcode 13.4.1. 🎉 

I am sorry to read @mohit-kapadia-at-lirisoft that it didn't work for you.  😞
